crypto/nrf_cc310_mbedcrypto/include/nrf-config-cc310.h

```diff
@@ -1318,6 +1346,44 @@
  */
 //#define MBEDTLS_PSA_CRYPTO_DRIVERS
 
+/** \def MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG
+ *
+ * Make the PSA Crypto module use an external random generator provided
+ * by a driver, instead of Mbed TLS's entropy and DRBG modules.
```

Comment:
Inconsistency: Here it's `Mbed TLS`, other places it's `mbed TLS`.

Reply:
should be lowercase

Reply:
This is from the original configuration file of the upstream configuration so I will not update it here. I will check if it is solved in later versions if not I can open a PR upstream.